TLM/TLM/UI/Localization/Translation.cs

```diff
@@ -152,7 +148,6 @@ public class Translation {
             guidesLookup_ = new Localization.LookupTable("Guide");
```

Comment:
I thought that filed name should correspond with LookupTable and filename, @kvakvs am I right?

Reply:
yes. I wanted it to be `Guides.csv` like `tutorials.csv` but I made a mistake. its too late for that now.

Reply:
there i changed them to guide.

Reply:
> yes. I wanted it to be `Guides.csv` like `tutorials.csv` but I made a mistake. its too late for that now.

According to PR description:
> NEW FILE : Guide.csv

🙂